packages/vscode-extension/src/handlers.ts

```diff
@@ -1155,6 +1155,39 @@ async function openMarkdownHandler() {
   }
 }
 
+export async function openReadMeHandler(args: any[]) {
+  if (workspace.workspaceFolders && workspace.workspaceFolders.length > 0) {
```

Comment:
What if current project is not a valid teamsfx project?

Reply:
this PR is only for happy path. For this case, it is still under discussion. there should be other PR to take care this case.